recipes/htmlmin/meta.yaml

```diff
@@ -0,0 +1,36 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set version = "0.1.10" %}
+
+package:
+    name: htmlmin
+    version: {{ version }}
+
+source:
+    fn: htmlmin-{{ version }}.tar.gz
+    url: https://files.pythonhosted.org/packages/60/de/f42baab5b8cd8834311dfccf90bd7275bd8bab0f82780e244a5b05a165c1/htmlmin-0.1.10.tar.gz
+    sha256: ca5c5dfbb0fa58562e5cbc8dc026047f6cb431d4333504b11853853be448aa63
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+    run:
+        - python
+
+test:
+    imports:
+        - htmlmin
+
+about:
+    home: https://htmlmin.readthedocs.org/en/latest/
+    license: BSD
```

Comment:
It seems that the license is `BSD 3-Clause`. Can you confirm that and update it here?